### PR TITLE
[AEA-1420] [AEA-1424] [AEA-1415] [AEA-1414] Fix/misc coverage tests

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

coverage fixes

## Fixes

fixes for cli remove command #2360
fix for mam plugins load after dependencies installed

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/multiplexer.py [hunk 1]
```diff
@@ -342,9 +342,9 @@ async def disconnect(self) -> None:
                 self.connection_status.set(ConnectionStates.disconnecting)
                 await asyncio.wait_for(self._stop(), timeout=60)
                 self.logger.debug("Multiplexer disconnected.")
-            except (CancelledError) as e:
-                self.logger.exception("Exception on disconnect:")
-                raise AEAConnectionError("Failed to disconnect the multiplexer.") from e
+            except CancelledError:  # pragma: nocover
+                self.logger.debug("Multiplexer.disconnect cancellation!")
+                raise
```
Comment (DavidMinarsch) [inline aea/multiplexer.py:347]: What problem does this relate to? Looks good, just wondering

**Review by DavidMinarsch — APPROVED** [review]
LGTM

File: aea/crypto/plugin.py [hunk 2]
```diff
@@ -129,7 +129,8 @@ def _get_plugins(group: str) -> List[Plugin]:
     :param group: the plugin group.
     :return: a mapping from plugin name to Plugin objects.
     """
-    entry_points: List[EntryPoint] = list(iter_entry_points(group=group))
+
+    entry_points: List[EntryPoint] = list(WorkingSet().iter_entry_points(group=group))
```
Comment (marcofavorito) [inline aea/crypto/plugin.py:133]: what is the difference?

Comment (solarw) [reply aea/crypto/plugin.py:133]: `WorkingSet()` so looks like default workingset populated once, and iter_entry_points is the same during python process lifetime, even you added removed python packages.

File: aea/multiplexer.py (hunk 1 above)
Comment (solarw) [reply aea/multiplexer.py:347]: not a problem, just log that disconnect cancelled and exception propagated,
it's not assumed that disconnect should be cancelled, but it's possible

**Pull request merged**